advocacy_docs/pg_extensions/pg_squeeze/using.mdx

```diff
@@ -3,9 +3,34 @@ title: Using pg_squeeze
 navTitle: Using
 ---
 
+The following sections explain how to use `pg_squeeze`.
```

Comment:
Delete.

Reply:
If you want some intro text (to fix the vertical space issue), then try something like, You use `pg_squeeze` to [accomplish whatever].